gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
> service/google is too long 

When you call it, it's only `google` (3 words vs. 5 words). Long?

> service name is more appropriate than company name for package name.

`gcm` is no longer service name that package supports. 

Using `google` is not that bad practice 

https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/config.go#L1